sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,167 @@
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow virtiofsd to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* You might encounter an issue where `machine-config-operator` (MCO) pod changes to `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the POD shows "sandboxed-containers-operator-scc" instead of the default `hostmount-anyuid`.
++
+If this happens, temporarily change the `seLinuxOptions` strategy in the `sandboxed-containers-operator-scc` SCC to the less restrictive `RunAsAny`, so that the admission process does not prefer it over the `hostmount-anyuid` SCC. You can do this by running the following command:
++
+. Change the `seLinuxOptions` strategy by running the following command:
++
+[source,terminal]
+----
+oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "RunAsAny"}}'
+----
+
+. Restart the MCO POD by running the following commands:
++
+[source,terminal]
+----
+oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=0
+----
++
+[source,terminal]
+----
+oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=1
+----
+
+. Revert the 'seLinuxOptions' strategy of the `sandboxed-containers-operator-scc` to its original value of 'MustRunAs' by running the following command:
++
+[source,terminal]
+----
+oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "MustRunAs"}}'
+----
+
+. Verify that the `hostmount-anyuid` SCC is applied to the `machine-config-operator` POD by running the following command:
++
+[source,terminal]
+----
+oc get pods -n openshift-machine-config-operator -l k8s-app=machine-config-operator -o yaml | grep scc
+openshift.io/scc: hostmount-anyuid
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2057545[*BZ#2057545*])
+
+* The {sandboxed-containers-operator} pods that utilize container CPU resource limits to increase the number of available CPUs for the pod may receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resources using `oc rsh <pod>` and running `lscpu`.
++
+[source,terminal]
+----
+# lscpu
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
```

Comment:
If this is a command (`# lscpu`) and example output, it should be separated into separate code blocks.